app/scripts/modules/core/src/presentation/less/imports/mixins.less

```diff
@@ -14,12 +14,12 @@
   float: left !important;
 }
 
-.fade-in(@duration:0.25; @delay: 0) {
+.fade-in(@duration:0.25s; @delay: 0) {
```

Comment:
obviously unrelated to the subject of the PR but I forgot I had this sitting in my dock and I'm not removing it.